x/launch/keeper/request.go

```diff
@@ -131,32 +132,62 @@ func CheckAccount(ctx sdk.Context, k Keeper, launchID uint64, address string) (b
 func ApplyRequest(
 	ctx sdk.Context,
 	k Keeper,
-	launchID uint64,
+	chain types.Chain,
 	request types.Request,
+	coord profiletypes.Coordinator,
 ) error {
-	if err := CheckRequest(ctx, k, launchID, request); err != nil {
+	err := CheckRequest(ctx, k, chain.LaunchID, request)
+	if err != nil {
 		return err
 	}
 
 	switch requestContent := request.Content.Content.(type) {
 	case *types.RequestContent_GenesisAccount:
 		ga := requestContent.GenesisAccount
 		k.SetGenesisAccount(ctx, *ga)
+		err = ctx.EventManager().EmitTypedEvent(&types.EventGenesisAccountAdded{
+			GenesisAccount: *ga,
+			LaunchID:       chain.LaunchID,
+		})
+
 	case *types.RequestContent_VestingAccount:
 		va := requestContent.VestingAccount
 		k.SetVestingAccount(ctx, *va)
+		err = ctx.EventManager().EmitTypedEvent(&types.EventVestingAccountAdded{
+			VestingAccount: *va,
+			LaunchID:       chain.LaunchID,
+		})
+
 	case *types.RequestContent_AccountRemoval:
 		ar := requestContent.AccountRemoval
-		k.RemoveGenesisAccount(ctx, launchID, ar.Address)
-		k.RemoveVestingAccount(ctx, launchID, ar.Address)
+		k.RemoveGenesisAccount(ctx, chain.LaunchID, ar.Address)
+		k.RemoveVestingAccount(ctx, chain.LaunchID, ar.Address)
+		err = ctx.EventManager().EmitTypedEvent(&types.EventAccountRemoved{
+			Address:  ar.Address,
+			LaunchID: chain.LaunchID,
+		})
+
 	case *types.RequestContent_GenesisValidator:
 		ga := requestContent.GenesisValidator
 		k.SetGenesisValidator(ctx, *ga)
+		err = ctx.EventManager().EmitTypedEvent(&types.EventValidatorAdded{
+			GenesisValidator:   *ga,
+			LaunchID:           chain.LaunchID,
+			HasCampaign:        chain.HasCampaign,
+			CampaignID:         chain.CampaignID,
+			CoordinatorAddress: coord.Address,
```

Comment:
I think we can have `CoordinatorAddress` for every request applied event for consistency